test/integration/packwerk/custom_executable_integration_test.rb

```diff
@@ -128,6 +128,33 @@ class CustomExecutableIntegrationTest < Minitest::Test
       end
     end
 
+    test "'packwerk check' does not blow up when parsing files with syntax issues from a false positive association" do
+      open_app_file(TIMELINE_PATH.join("app", "models", "bad_file.rb")) do |file|
+        # This is an example of a file that references `belongs_to`, but as an ActiveAdmin API
+        # rather than as a Rails Association concept
+        content = <<~CONTENT
+          require 'rails_helper'
+
+          RSpec.describe ActiveAdmin::Resource::BelongsTo do
+            it "should have an owner" do
+              expect(belongs_to.owner).to eq post_config
+            end
+          end
+        CONTENT
+
+        file.write(content)
+        file.flush
+
+        refute_successful_run("check")
+
+        assert_match(/Packwerk is inspecting 13 files/, captured_output)
+        assert_match(%r{components/timeline/app/models/bad_file.rb}, captured_output)
+        assert_match(/Passed `nil` into T.must/, captured_output)
```

Comment:
Should we be making this code more fault tolerant? If this stack trace is within Packwerk, I imagine the end user should not be concerned with this and it is actually a bug.